docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -0,0 +1,61 @@
+# ECE340
+
```

Comment:
https://github.com/hkn-alpha/wiki/pull/6#discussion_r1251461268
Add sth like this

Reply:
For this, just add a sentence about how this is required for EE and a 1-of-6 for CE.